docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,105 @@
+{% extends "guides/insar_product_guide_template.md" %}
+
+{% block header %}
+# Sentinel-1 Burst InSAR Product Guide
+
+TODO
+{% endblock %}
+
+{% block processing_options %}
+### Processing Options
+TODO
+
+{% endblock %}
+
+{% block workflow %}
+## InSAR Workflow
+TODO
+
+### Pre-Processing
+
+Pre-processing steps prepare the SAR images to be used in interferometry.
+The pre-processing steps include downloading the burst granules,
+downloading the DEM file,
+and downloading the orbit and auxiliary data files.
+
+#### Download Bursts
+
+The burst InSAR workflow accepts as input two
+[Interferometric Wide swath Single Look Complex](https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/acquisition-modes/interferometric-wide-swath "https://sentinel.esa.int/web/sentinel/user-guides/sentinel-1-sar/acquisition-modes/interferometric-wide-swath" ){target=_blank}
+(IW SLC) burst granules with the same burst ID.
+The bursts are downloaded using ASF's
+[Sentinel-1 Burst Extractor](https://sentinel1-burst-documentation.asf.alaska.edu/ "https://sentinel1-burst-documentation.asf.alaska.edu/" ){target=_blank}.
+
+#### Download the DEM File
+
+In order to create differential InSAR products that show motion on the ground,
+one must subtract the topographic phase from the interferogram.
+The topographic phase, in this case, is replicated by using an
+[existing DEM](../dems.md "HyP3 DEM Documentation" ){target=_blank}
+to calculate the actual topographic phase.
+This phase is then removed from the interferogram leaving just the motion or deformation signal
+(plus atmospheric delays and noise).
+
+The DEM that is used for HyP3 InSAR processing is the
+[2021 Release of the Copernicus GLO-30 Public DEM](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model "Copernicus DEM" ){target=_blank}
+dataset
+[publicly available on AWS](https://registry.opendata.aws/copernicus-dem/ "https://registry.opendata.aws/copernicus-dem" ){target=_blank},
+which provides global coverage at 30-m pixel spacing.
+For more information about the 2021 updates, see the 'Releases' section of
+[this article](https://spacedata.copernicus.eu/collections/copernicus-digital-elevation-model "Copernicus DEM" ){target=_blank}.
+
+The portion of the DEM that covers the input bursts is downloaded.
+
+#### Download Orbit Files and Calibration Auxiliary Data Files
+
+The orbit files are downloaded from the
+[Copernicus Sentinels POD Data Hub](https://scihub.copernicus.eu/gnss/#/home "Copernicus Sentinels POD Data Hub" ){target=_blank}.
+
+The calibration auxiliary data files are downloaded from the
+[Sentinel-1 Mission Performance Center](https://sar-mpc.eu/ "Sentinel-1 Mission Performance Center" ){target=_blank}.
+
+TODO: add more detail? i.e. what the files are and what they're used for
```

Comment:
@hjkristenson Feel free to add some more detail regarding the orbit and auxcal files if you're interested, otherwise we could probably just leave it as-is for now.

Reply:
I'll pass this responsibility along to @forrestfwilliams or @asjohnston-asf or anyone else who's actually worked with the orbit/calibration files.